assets/src/stories-editor/blocks/amp-story-page/edit.js

```diff
@@ -142,10 +149,30 @@ const PageEdit = ( {
 		}
 	}, [ mediaType, mediaUrl ] );
 
+	// If there is more than one immovable block, leave only the last and remove the others.
+	useEffect( () => {
+		const immovableBlocks = getImmovableBlocks( clientId );
+		if ( immovableBlocks.length > 1 ) {
+			immovableBlocks.pop();
+			const blocksToRemove = immovableBlocks.map( ( { clientId: blockId } ) => blockId );
+			removeBlocks( blocksToRemove );
+			createErrorNotice(
+				__( 'Block removed. Only one CTA/Attachment block allowed per Page.', 'amp' ),
```

Comment:
Add count here - technically there could have been more than 1 block removed. Not probable under current circumstances, but to be safe.

Reply:
We know which block was removed here. Just say CTA or attachment. Much clearer

Reply:
Count added in 58b22d6.